integration/integration-end-to-end.spec.ts

```diff
@@ -133,7 +150,14 @@ test("Upload configuration file, choose form, fill form, submit form correctly",
   await t.click(SubmitButton);
 
   // Check that EBL is created
+  const fileName = "Document-1-local.tt";
   await t.expect(Title.textContent).contains("Document(s) issued successfully");
-  await t.expect(Selector("div").withText("Document-1-local.tt").exists).ok();
+  await t.expect(Selector("div").withText(fileName).exists).ok();
   await t.expect(Selector("div").withText("Download").exists).ok();
+
+  // Check that the file has been downloaded successfully
+  await t.click(downloadLink);
+  const filePath = `${downloadsFolder()}/${fileName}`;
+  t.ctx.filePath = filePath; // For use in cleanup
+  await t.expect(await waitForFileDownload(t, filePath)).eql(true);
```

Comment:
Improving test case coverage by actually downloading the file, and checking if it's successfully downloaded.

A better case would be to then check the file contents. However, it's tricky because the values are non-deterministic because of the hash.

We can leave that for another PR, if ever needed.

Reply:
This will be quite similar to just checking if there is a `<a>` with href with data in it right?

Reply:
Can use JS to check the data file, can run a getData on it so that it's deterministic, probably can skip the oa-verify part though.

Reply:
It seems like for file download, the `<a>` with `href` approach is no longer used in favour of `saveAs`. I'll figure how to grab the contents.